### PR TITLE
feat(nasha): add onboarding page

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nasha-refresh`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8913
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

image

### CODE REVIEW

File: packages/manager/modules/nasha/src/nasha.routing.js [hunk 1]
```diff
@@ -8,6 +8,16 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
     resolve: {
       breadcrumb: () => NASHA_TITLE,
     },
+    redirectTo: (transition) =>
```
Comment (JeremyDec) [inline packages/manager/modules/nasha/src/nasha.routing.js:11]: As mentioned by Sonarcloud (see: https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=6788&id=ovh_manager&open=AX_cpCPP2APkB2szqB4R), redirectTo is already defined.  Should it be replaced?

**Review by JeremyDec — CHANGES_REQUESTED** [review]

File: packages/manager/modules/nasha/src/nasha.routing.js (hunk 1 above)
Comment (y4nnL) [reply packages/manager/modules/nasha/src/nasha.routing.js:10]: My bad, merge issue... Fixed :+1:

**Review by frenautvh — APPROVED** [review]

File: packages/manager/modules/nasha/src/nasha.routing.js [hunk 2]
```diff
@@ -3,11 +3,20 @@ import { NASHA_TITLE } from './nasha.constants';
...
     template: '<div ui-view></div>',
     resolve: {
       breadcrumb: () => NASHA_TITLE,
     },
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .get('iceberg')('/dedicated/nasha')
+        .query()
+        .limit(1)
```
Comment (radireddy) [inline packages/manager/modules/nasha/src/nasha.routing.js:15]: 👍

File: packages/manager/modules/nasha/src/onboarding/onboarding.controller.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+export default class NashaOnboardingController {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    const { ovhSubsidiary } = coreConfig.getUser();
+    this.illustration = illustration;
+    this.guides = GUIDES.map((guide) => ({
+      ...guide,
+      title: $translate.instant(`nasha_onboarding_${guide.id}_title`),
+      description: $translate.instant(`nasha_onboarding_${guide.id}_content`),
+      link: guide.link[ovhSubsidiary] || guide.link.FR,
```
Comment (radireddy) [inline packages/manager/modules/nasha/src/onboarding/onboarding.controller.js:13]: default should be English right? can be OVH world or UK?

File: packages/manager/modules/nasha/src/onboarding/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "nasha_onboarding_title": "NAS-HA",
```
Comment (radireddy) [inline packages/manager/modules/nasha/src/onboarding/translations/Messages_fr_FR.json:2]: can be removed from translation. Hard code the value directly.

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**